includes/class-civicrm-caldera-forms-helper.php

```diff
@@ -484,13 +484,43 @@ public function map_fields_to_prerender( $config, &$form, $ignore_fields, $entit
 				if ( $field['type'] == 'radio' ) {
 					$options = Caldera_Forms_Field_Util::find_option_values( $field );
 					$form['fields'][$field['ID']]['config']['default'] = array_search( $value, $options );
+				} elseif ( $field['type'] == 'date_picker' ) {
+					// If the date is still in the format returned by the CiviCRM API, convert it to the date picker's format.
+					$filtered_value = &$form['fields'][$field['ID']]['config']['default'];
+					if ( preg_match('/^\d{4}-\d{2}-\d{2}(?: \d{2}:\d{2}:\d{2})?$/', $filtered_value ) ) {
+						$format = self::translate_date_picker_format( $field['config']['format'] );
+						$filtered_value = date_create( $filtered_value )->format( $format );
+					}
 				}
 			}
 		}
 
 		return $form;
 	}
 
+	/**
+	 * Translate Caldera Forms date picker formats to PHP date formats.
+	 *
+	 * @since 1.0.2
+	 *
+	 * @param string $date_picker_format The Caldera Forms date picker format
+	 */
+	public static function translate_date_picker_format( $date_picker_format ){
+		// Translate each token used in the CF date picker format to the corresponding PHP format character.
+		static $token_map = [
```

Comment:
No need for static here.

Reply:
Again, there's no _need_ for the static here, but I put it in as I thought it would be slightly more efficient: The variable will only be instantiated once instead of every time the function is called. But again, it's your repo so I'll remove it if you prefer.

Reply:
Unless necessary, I prefer no statics :)